PerfectWedding/src/main/java/com/infoshareacademy/ProvidersEdit.java

```diff
@@ -61,6 +61,7 @@ public void editSelectedProvider(int providerId) {
                     break;
                 case "location":
                     providersList.get(providerId).setLocation(new Location(scanInput("Podaj miejscowosc: ")));
```

Comment:
Tu trzeba raczej zrezygnować z linii 63 - wywoływana jest w niej funkcja, która przyjmuje ze scannera miejscowość, którą podajemy w konsoli natomiast metoda setVoivodeship również to robi.